rclcpp/include/rclcpp/parameter_client.hpp

```diff
@@ -154,6 +157,28 @@ class AsyncParametersClient
       void(std::shared_future<rcl_interfaces::msg::SetParametersResult>)
     > callback = nullptr);
 
+  /**
+   * @brief delete_parameters allows to delete several parameters
+   * like "ros2 param delete" would do
+   * @param parameters_names vector of parameters names
+   * @return the future of the set_parameter service used to delete the parameters
+   */
```

Comment:
Please put the brief on a separate line with `///` and use `\` rather than `@`, in order to match this project's style. Also try to use imperative syntax in the brief (as I do below), and please use complete sentences. Putting it all together, e.g.:

```suggestion
  /// Delete several parameters at once.
  /**
   * This functions like command-line tool `ros2 param delete` would.
   *
   * \param parameters_names vector of parameters names
   * \return the future of the set_parameter service used to delete the parameters
   */
```